greenboot.spec

```diff
@@ -54,12 +66,13 @@ install -Dpm 0755 etc/greenboot/check/required.d/00_required_scripts_start.sh %{
 mkdir    %{buildroot}%{_sysconfdir}/%{name}/check/wanted.d
 install -Dpm 0755 etc/greenboot/check/wanted.d/00_wanted_scripts_start.sh %{buildroot}%{_sysconfdir}/%{name}/check/wanted.d/00_wanted_scripts_start.sh
 mkdir    %{buildroot}%{_sysconfdir}/%{name}/green.d
-install -Dpm 0755 etc/greenboot/green.d/00_grub_fallback.sh %{buildroot}%{_sysconfdir}/%{name}/green.d/00_grub_fallback.sh
 install -Dpm 0755 etc/greenboot/green.d/00_greenboot_notification.sh %{buildroot}%{_sysconfdir}/%{name}/green.d/00_greenboot_notification.sh
-install -Dpm 0755 etc/greenboot/green.d/01_greenboot_motd.sh %{buildroot}%{_sysconfdir}/%{name}/green.d/01_greenboot_motd.sh
+install -Dpm 0755 etc/greenboot/green.d/01_ostree_grub2_fallback.sh %{buildroot}%{_sysconfdir}/%{name}/green.d/01_ostree_grub2_fallback.sh
+install -Dpm 0755 etc/greenboot/green.d/02_grub2_boot_success.sh %{buildroot}%{_sysconfdir}/%{name}/green.d/02_grub2_boot_success.sh
+install -Dpm 0755 etc/greenboot/green.d/50_greenboot_motd.sh %{buildroot}%{_sysconfdir}/%{name}/green.d/50_greenboot_motd.sh
```

Comment:
You should be able to collapse these to something like:

```
install -Dpm 0755 etc/greenboot/green.d/* %{buildroot}%{_sysconfdir}/%{name}/green.d
```

And similar for other dirs.